web/viewer.css

```diff
@@ -1104,11 +1104,15 @@ a.secondaryToolbarButton[href="#"] {
   -moz-appearance: textfield; /* hides the spinner in moz */
   text-align: right;
   width: 40px;
+  background-size: 0 0;
+  transition-property: background-size;
```

Comment:
Similar to the comment above, could we not use the existing `background-image`-property instead, since using `background-size` feels like a hack?

In any case, a good comment explaining what's happening here seems necessary to aid future readers (that might even be us in a few months time).

Reply:
Making the transition on `background-image` just doesn't work in Firefox. We can't change the visibility as above because this is the background of an `input` so the only workaround I found is to make the transition on `background-size`.